src/Elements/types.jl

```diff
@@ -0,0 +1,180 @@
+const HEADERS   = ["#", "##", "###", "####", "#####", "######"]
+const EMPHASIS  = ["*", "**"]
+
+const CATEGORIES = [:module,    :function, :method, :macro, :type,
```

Comment:
I'm thinking we'll probably just merge ``:function`` and ``:method`` in *all* output. Will help out the "long signatures" problem.